airbyte-webapp/src/components/ui/PillButton/PillButton.module.scss

```diff
@@ -0,0 +1,19 @@
+@use "scss/colors";
+
+.button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 3px 10px;
+  gap: 5px;
+  border-radius: 20px;
```

Comment:
is this worth adding a border-radius variable?  do we anticipate reusing this shape?

Reply:
Still need to finalize the border radius variables. I'll follow up with Nico and apply this later on.